CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## v1209
+
+* Staging release date: TBD
+* Production APM-injected release date: TBD
+* Production Standalone release date: TBD
+
+
+### Doubled the limit of PageAction events per harvest
+Up to 120 PageAction events can be harvested every 30 seconds.
+
+### Prevent duplicate session trace nodes
+The final Session Trace node in a harvest, captured using the Resource Timing API, is no longer duplicated in the subsequent harvest.
+
+This issue lead to 1 duplicate node in a Session Trace, every 10 seconds, over the duration of the trace.
+
+### Memory overhead when agent script 
+Fixed a memory leak in the agent when the network request to load the second part of the agent is blocked.
+
+### Update to file protocol restriction
+Fixed an error thrown in the console when the agent is loaded using the `file://` protocol. Caused by features in the agent trying to run when others had been aborted.
```

Comment:
I would combine into a single sentence, or start the second sentence with `This was caused...` :)